Move NFT and fractional-NFT to archive folder with warnings (#712)
FuelLabs/sway-applications: archived_apps/fractional-NFT/README.md

    </a>
</p>

> **Warning**

> > This application have been temporarily archived waiting the finalised release of of the [Sway multi-token standard](https://github.com/FuelLabs/rfcs/pull/17) before being updated. In it's current form, it does not comply with this standard and as such should be considered outdated.

Reviewer: ```suggestion
> This application have been temporarily archived until the release of the [Sway multi-token standard](https://github.com/FuelLabs/rfcs/pull/17). In its current form, it does not comply with this standard and as such should be considered outdated.
```

> > It is not advised to use this applciation for production purposes, use at your own risk

Reviewer: ```suggestion
> It is inadvisable to use this application for production purposes.
```
